scripts/customer-info/README.md

```diff
@@ -0,0 +1,74 @@
+# GENERAL SUMMARY
+On a high level, our goal is to improve the customer experience by allowing us to obtain 
+the introduction customer information faster. Through running this script, we will recieve 
+a yaml file that contains the targets, relevant information about actions (number of total 
+actions, set of mnemonics, set of configurations, set of platforms, and set of aspects), the 
+bazel version being used, and the values of the relevant bazel flags. 
+
+# RUNNING/EDITING THE SCRIPT 
+To run the script, 
+1. Navigate to the ```scripts``` folder and then the ```customer-info``` folder
+2. run command 
+   ```
+   python customer_info.py PATH_TO_REPO
+   ``` 
+   where ```PATH_TO_REPO``` = 
+   absolute path to repo
+   ```
+   python customer_info.py "/Users/sarahraza/example"
+   ```
+   
+To customize the script,
+- Change the list of relevant flags by editing the ```relevant_flags``` variable 
```

Comment:
Ideally, the end user should not be changing the list of flags here. We will finalize what this list should look like before making the script available to folks. 

In the future, we'll update the list as we come across new flags relevant to remote caching/execution.

Reply:
Sounds good! Do you think this would be better as an argument that's passed in?